packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1266,6 +1296,8 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
   };
 }
 
+GraphiQLInternals.contextType = MigrationContext;
```

Comment:
This is the magic that enables `this.context.schema` inside the class component.

Reply:
oh yes, i remember it used to work this way with the old context API too, amazing how little the class component interface changed